go.mod

```diff
@@ -21,3 +21,5 @@ require (
 	sigs.k8s.io/controller-tools v0.2.9-0.20200331153640-3c5446d407dd
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/openshift/machine-api-operator v0.2.1-0.20200527204437-14e5e0c7d862 => github.com/Danil-Grigorev/machine-api-operator v0.2.1-0.20200604073207-04068f243c6b
```

Comment:
I'd prefer this to be moved back to master openshift before merge
Hold until we can identify whether this is absolutely necessary
/hold